### DISCUSSION TITLE
Read arrays of type string

### DISCUSSION THREAD

**Discussion by jfw96** [toml-f/toml-f#140] [Q&A]
Is there currently a way to read in arrays of type string? I have not found a corresponding overload of get_value and wonder if there is a special reason for this.

**Comment by awvwgk** [comment]
Array of characters are a special case in Fortran since all values need to have the same length. There would be the option to either pad with spaces (which requires usage of trim a lot) or to introduce a derived type to represent strings. Both seem suboptimal in their own way.
